salt/metalk8s/salt/master/installed.sls

```diff
@@ -36,6 +36,13 @@ Install and start salt master manifest:
       - file: /etc/salt/master.d/99-metalk8s.conf
       - file: /etc/salt/master.d/99-metalk8s-roots.conf
 
+Delay after new pod deployment:
+  module.wait:
+    - test.sleep:
+      - length: 10
+    - watch:
+      - metalk8s: Install and start salt master manifest
```

Comment:
You need to require this in `Make sure salt master container is up` and also (I know was already here before) require `Make sure salt master container is up` in `Wait for Salt API to answer`